### DISCUSSION TITLE
lotus lite node network

### DISCUSSION THREAD

**Discussion by Alihszh** [filecoin-project/lotus#9842] [Build and Sync] [answered]
Hello everyone, I hope all is well.
I was able to set up the lotus lite node by following the instructions, however I am unable to interact with filecoin storage since I am unable to obtain file coin through a faucet.
I used "lotus wallet new" to create a wallet address, tried to obtain some faucet money from calibration, and the transaction was successful. However, when I checked "lotus wallet list" on the Lotus Lite Node, it showed that I had no filecoin.
My network may not be on calibration, but I had no idea how to check for it or how to switch to calibration with a Lotus Lite Node.
I'd appreciate your opinion.

**Comment by TippyFlitsUK** [comment]
What is your output for `lotus version` or` lotus -v`?

**Reply by Alihszh** [reply]
> What is your output for `lotus version` or` lotus -v`?
lotus version 1.18.1+calibnet+git.e68b8e96c

**Reply by Alihszh** [reply]
I updated it it's now this:
Daemon:  1.18.1+calibnet+git.e68b8e96c+api1.5.0
Local: lotus version 1.18.1+calibnet+git.e68b8e96c

**Comment by Alihszh** [comment, marked as answer]
Following some testing, I discovered that I should sync my node using `lotus sync wait `at which point my node would be updated and I would be able to access my test FIL on the wallet address.
